src/harmony-rpc.ts

```diff
@@ -1,16 +1,18 @@
-import { getStatic } from '@ethersproject/properties'
+import { BigNumber } from '@ethersproject/bignumber'
```

Comment:
let's change this and the spec file name to `HRC721`, the `base-hr721` to `base-HRC721`